docs/developers.md

```diff
@@ -34,8 +34,7 @@ To run Helm locally, you can run `bin/helm`.
 
 ### Man pages
 
-Man pages and Markdown documentation are already pre-built in `docs/`. You may
-regenerate documentation using `make docs`.
+Man pages and Markdown documentation are already pre-built in `docs/`.
```

Comment:
This should be flipped to something like... Docs are not pre-built but you can generate them using `make docs`.

Reply:
Done